.travis.yml

```diff
@@ -16,8 +16,6 @@ matrix:
       python: 3.6
     - env: TOXENV=docs
       python: 3.6
-    - env: TOXENV=py27
-      python: 2.7
```

Comment:
Although I'm afraid we will lose users in the short-term. A lot of people in the lab still use Python 2.7

Reply:
Py3 is just too convenient! It will make good for them too 😛